moped-editor/src/views/projects/projectView/ProjectComponentsMapView.js

```diff
@@ -110,17 +271,35 @@ const ProjectComponentsMapView = ({
         onHover={handleLayerHover}
         /* Updates state of viewport on zoom, scroll, and other events */
         onViewportChange={handleViewportChange}
+        mapStyle={mapStyleConfig}
       >
         {/* Draw Navigation controls with specific styles */}
         <div className={classes.navStyle}>
           <NavigationControl showCompass={false} />
         </div>
+
+        {/* GEOCODER */}
+        <Geocoder
+          mapRef={mapRef}
+          onViewportChange={handleGeocoderViewportChange}
+          mapboxApiAccessToken={MAPBOX_TOKEN}
+          bbox={mapConfig.geocoderBbox}
+          containerRef={mapGeocoderContainerRef}
+          marker={false}
+          position="top-right"
+        />
         {/*
           If there is GeoJSON data, create sources and layers for
           each source layer in the project's GeoJSON FeatureCollection
         */}
         {projectFeatureCollection &&
-          createSummaryMapLayers(projectFeatureCollection)}
+          createSummaryMapLayers({
+            type: "FeatureCollection",
+            features: projectFeatureCollection.features.filter(feature =>
+              visibleLayerIds.includes(feature?.properties?.sourceLayer)
+            ),
+          })}
+
```

Comment:
The filtering works a little different than the map, this is because we do not make use of interactive layers.